test/unit/singlePageApp.test.ts

```diff
@@ -55,24 +54,24 @@ describe("single page app", () => {
         expect(
             get(
                 cfTemplate.Resources[cfDistributionLogicalId],
-                "Properties.DistributionConfig.DefaultCacheBehavior.FunctionAssociations"
-            )
+                "Properties.DistributionConfig.DefaultCacheBehavior.FunctionAssociations",
+            ),
         ).toMatchInlineSnapshot(`
-            Array [
-              Object {
+            [
+              {
                 "EventType": "viewer-response",
-                "FunctionARN": Object {
-                  "Fn::GetAtt": Array [
-                    "${responseFunction}",
+                "FunctionARN": {
+                  "Fn::GetAtt": [
+                    "landingResponseFunctionA308C722",
```

Comment:
This needs to be the variable (`${responseFunction}`) not the hardcoded ID because the ID might change.